DigitalLearningSolutions.Data.Tests/DataServices/SupervisorDelegateDataServiceTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.DataServices
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Transactions;
+    using Dapper;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Email;

```

Comment:
`DigitalLearningSolutions.Data.Models.Email` is unused

Reply:
removed